pydoclint/main.py

```diff
@@ -142,6 +143,7 @@ def validateStyleValue(
         ' over the .toml file'
     ),
 )
+@click.version_option(__version__)
```

Comment:
A curious question: if I'd like to add a shorthand for `--version` (for example, `-v`), is it feasible?  I'm not an expert in using `click`.

(Although `-v` may not be a good idea because people may mistake it for "verbose".  So this is just a curious question of mine.)